libs/full/runtime_distributed/src/server/runtime_support_server.cpp

```diff
@@ -1046,7 +1046,12 @@ namespace hpx { namespace components { namespace server {
 
             // secondary command line handling, looking for --exit and other
             // options
-            std::string cmd_line(ini.get_entry("hpx.cmd_line", ""));
+            std::string cmd_line =
+                ini.get_entry("hpx.commandline.command", "") +
```

Comment:
I think there should be an extra space at least after the command because the options don't all contain the leading space.